### PR TITLE
Articles

### PR DESCRIPTION
**Pull request by dfischer4623**
Integration of API for articles and Index.html is completed in branch: articles.

### CODE REVIEW

**Comment by dfischer4623** [comment]
Please review.  I will merge or you can merge after your review.

File: index.html [hunk 1]
```diff
@@ -135,6 +137,9 @@ <h3>Latest news about <span>[populate name/industry/sector from the input form]<
         </section>
     </main>
 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (BohdiCave) [inline index.html]: I actually already included this link to jQuery on line 142/147 below, so you can just remove this line here.

File: index.html [hunk 2]
```diff
@@ -135,6 +137,9 @@ <h3>Latest news about <span>[populate name/industry/sector from the input form]<
...
+    <script src="articles.js"></script>
```
Comment (BohdiCave) [inline index.html]: Generally, as I understand, the suggestion is to call the scripts after all the HTML code - the last thing to do before </body> tag. If you move this link below, just below the jQuery link but before my own code I've included, I think that should work best. Eventually, I'll move my code to the .js file as well - once we figure out the full sequence of calls and functions.

File: index.html [hunk 3]
```diff
@@ -106,17 +106,19 @@ <h3>Stock performance data</h3>
...
-                <article class="column column-80 news-item" id="item02"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item03"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item04"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item05"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item06"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
+                <article class="column column-80 news-item" id="item00"><br><br><br><br></article>
+                <article class="column column-80 news-item" id="item01"><br><br><br><br></article>
+                <article class="column column-80 news-item" id="item02"><br><br><br><br></article>
+                <article class="column column-80 news-item" id="item03"><br><br><br><br></article>
```
Comment (BohdiCave) [inline index.html]: Maybe also remove the 4 <br> between the <article> tags. That was just a placeholder for the mock-up. Let's see how it loads and then make the styling changes necessary.

**Review by BohdiCave — CHANGES_REQUESTED** [review]
I just three brief comments, quick changes before merging. Mostly copy-paste stuff. I'll approve once you make them and we'll merge it in. Thanks!

**Review by BohdiCave — APPROVED** [review]

**Pull request merged**